compass/landice/tests/antarctica/mesh.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, test_case):
             The test case this step belongs to
 
         """
-        super().__init__(test_case=test_case, name='mesh', cpus_per_task=64,
+        super().__init__(test_case=test_case, name='mesh', cpus_per_task=128,
```

Comment:
I changed this to 128 cores for consistency with other steps.  I think it would be odd for the biggest mesh to use the fewest cores for mask generation and I think the Antarctic mesh can almost certainly use a full node worth of cores.